xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/PanelSheet.xml

```diff
@@ -53,14 +53,21 @@
 #set ($obj = $doc.getObject('Panels.PanelClass'))
 #if ($obj)
   #set($class = $obj.xWikiClass)
-  #displayPanelProperty($obj 'name')
-  #displayPanelProperty($obj 'type')
-  #displayPanelProperty($obj 'category')
-  #displayPanelProperty($obj 'description')
-  #displayPanelProperty($obj 'content')
-  #displayPanelProperty($obj 'async_enabled')
-  #displayPanelProperty($obj 'async_cached')
-  #displayPanelProperty($obj 'async_context')
+  ## Enclose in .xform class div when editing
+  #if ($xcontext.action == 'edit')
```

Comment:
.xform can also be in view mode, so you can remove the edit limitation. For example, if you look at the page "xwiki/bin/view/Help/Applications/Contributors/Frank%20Capra" from your local/test instance you will see how .xform behaves in view and edit mode.

Reply:
Yes. It's quite different. I think there are certain problems with the macro displayPanelProperty as it is defined.
Can I follow the Contributors page template for the displayPanelProperty macro? That seems to generate a label tag within the <dt> tag in a better way.